clients/gtest/blas1_gtest.cpp

```diff
@@ -174,13 +175,24 @@ TEST_P(blas1_gtest, iamax_float)
     Arguments arg = setup_blas1_arguments( GetParam() );
     rocblas_status status = testing_iamax<float>( arg );
     // if not success, then the input argument is problematic, so detect the error message
-    if(status != rocblas_status_success){
-        if( arg.N < 0 ){
-            EXPECT_EQ(rocblas_status_invalid_size, status);
-        }
-        else if( arg.incx < 0){
-            EXPECT_EQ(rocblas_status_invalid_size, status);
-        }
+    if(status != rocblas_status_success)
+    {
+        EXPECT_EQ(rocblas_status_success, status);
+    }
+}
+
+TEST_P(blas1_gtest, iamax_double)
+{
+    // GetParam return a tuple. Tee setup routine unpack the tuple
+    // and initializes arg(Arguments) which will be passed to testing routine
+    // The Arguments data struture have physical meaning associated.
+    // while the tuple is non-intuitive.
+    Arguments arg = setup_blas1_arguments( GetParam() );
+    rocblas_status status = testing_iamax<double>( arg );
+    // if not success, then the input argument is problematic, so detect the error message
+    if(status != rocblas_status_success)
+    {
+        EXPECT_EQ(rocblas_status_success, status);
```

Comment:
Do we need to protect the EXPECT_EQ with a branch?

Reply:
Possibly, you meant to use your new macro CHECK_ROCBLAS_ERROR()

Reply:
Thanks, you are correct, will remove branch and use CHECK_ROCBLAS_ERROR()